app/views/govuk_publishing_components/components/_radio.html.erb

```diff
@@ -71,6 +71,10 @@
 
             data_attrs = { "aria-controls": conditional_id }
             data_attrs["tracking-url"] =  item[:url] if item.key?(:url)
+
+            if item.fetch(:data_attributes, {}).any?
```

Comment:
Not a rails _connoisseur_, but can we use `local_assigns` instead? cc: @kevindew
I haven't seen this type of checking, it may be alright.

Reply:
It seems like we're using this approach in the govspeak component. Seems to be working, so I'll get over it.

Reply:
😆👍